### DISCUSSION TITLE
consulta sobre entregas

### DISCUSSION THREAD

**Discussion by ktyfer** [INGCOM-UNRN-P1/cursada-2023#118] [Preguntas y respuestas] [answered]
buenas! si hasta hoy no tengo un mail de revision del tp2 puede que por el momento este sin corregir o porque sale que no entregue?... porque del tp1 recibi mail a los 3 dias que lo entregue....

**Comment by martinvilu** [comment]
¿Que comisión?

**Reply by ktyfer** [reply]
comision 1

**Reply by martinvilu** [reply]
Le hacemos un ping a @inomdedeu

**Reply by martinvilu** [reply]
En las planillas compartidas, se ven 3 entregas con tu correo; si hay correcciones para hacer, tenes que completar el formulario denuevo.

Recordá que en total son ~100 por lo que tal vez tardemos en llegar.

**Reply by ktyfer** [reply]
ah ok vuelvo a enviar el formulario entonces?

**Reply by martinvilu** [reply, marked as answer]
si quedaban correcciones pendientes, yeap

**Reply by ktyfer** [reply]
ok gracias y en caso que tenga que reenviar un tp corregido tengo que enviar el formulario de nuevo?

**Reply by martinvilu** [reply]
si
